examples/create_log_entry.rs

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use openapi::apis::{configuration::Configuration, entries_api};
+use openapi::models::{
+    log_entry::LogEntry,
+    rekord::{AlgorithmKind, Data, Hash, PublicKey, Signature, Spec},
+    ProposedEntry,
+};
+use url::Url;
+
+#[tokio::main]
+async fn main() {
+    let configuration = Configuration::default();
+    let hash = Hash::new(
+        AlgorithmKind::sha256,
+        "e2535d638859bb63ea9ea5cf467562cba63b007eae1acd0d73a3f259c582561f".to_string(),
+    );
+    let data = Data::new(
+        hash,
+        Url::parse(
+            "https://raw.githubusercontent.com/jyotsna-penumaka/integrate-rekor/main/README.md",
+        )
+        .unwrap(),
+    );
+    let public_key = PublicKey::new(
+        "c3NoLWVkMjU1MTkgQUFBQUMzTnphQzFsWkRJMU5URTVBQUFBSVA3M2tuT0tKYVNyVEtEa2U2OEgvRlJoODRZWU5CU0tBN1hPVWRpWmJjeG8gdGVzdEByZWtvci5kZXYK".to_string(),
+    );
+    let signature = Signature::new(
+        "ssh".to_string(),
+        "LS0tLS1CRUdJTiBTU0ggU0lHTkFUVVJFLS0tLS0KVTFOSVUwbEhBQUFBQVFBQUFETUFBQUFMYzNOb0xXVmtNalUxTVRrQUFBQWcvdmVTYzRvbHBLdE1vT1I3cndmOFZHSHpoaApnMEZJb0R0YzVSMkpsdHpHZ0FBQUFFWm1sc1pRQUFBQUFBQUFBR2MyaGhOVEV5QUFBQVV3QUFBQXR6YzJndFpXUXlOVFV4Ck9RQUFBRURCYVFtVTRXNHFCQzBaM3N6aTZuSEE4cWlBdE5QVzFkU29UTmtlMTBOKzRRTUdad0pRMXR6QTVIYk5BUkxHc3cKN0I0b2RxWWFpRVEwSzMwdEtBZEcwSAotLS0tLUVORCBTU0ggU0lHTkFUVVJFLS0tLS0K".to_string(),
+        public_key,
+    );
+    let spec = Spec::new(signature, data);
+    let proposed_entry = ProposedEntry::Rekord {
+        api_version: "0.0.1".to_string(),
+        spec: spec,
+    };
+
+    let log_entry: LogEntry = entries_api::create_log_entry(&configuration, proposed_entry)
+        .await
+        .unwrap();
+    println!("{:#?}", log_entry);
+}
```

Comment:
I got an error result on this example:
```
[lily@localhost rekor-rs]$ cargo run --example create_log_entry
   Compiling openapi v0.0.1 (/home/lily/Repos/rekor-rs)
    Finished dev [unoptimized + debuginfo] target(s) in 2.24s
     Running `target/debug/examples/create_log_entry`
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ResponseError(ResponseContent { status: 409, content: "{\"code\":409,\"message\":\"An equivalent entry already exists in the transparency log with UUID b378b84b79477794b394025d1593a3a1480954c66d51c64ff75eed313cc9f922\"}\n", entity: Some(Status400(Error { code: Some(409), message: Some("An equivalent entry already exists in the transparency log with UUID b378b84b79477794b394025d1593a3a1480954c66d51c64ff75eed313cc9f922") })) })', examples/create_log_entry.rs:54:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

If this is expected, I would add some error handling here to make this error go away - and longer term in another PR we could make sure it can create a new log entry each time.